test/e2e/serial/tests/configuration.go

```diff
@@ -98,6 +97,10 @@ type mcpInfo struct {
 	sampleNode    corev1.Node
 }
 
+func (i mcpInfo) ToString() string {
+	return fmt.Sprintf("name %q; config %q; sample node %q\n", i.mcpObj.Name, i.initialConfig, i.sampleNode.Name)
```

Comment:
Stringer functions, either `String()` or our custom `ToString()` lesser format should never add the trailing "\n"